src/veh_interact.cpp

```diff
@@ -1729,6 +1733,10 @@ void veh_interact::display_details(const vpart_info &part)
 
     wborder(w_details, LINE_XOXO, LINE_XOXO, LINE_OXOX, LINE_OXOX, LINE_OXXO, LINE_OOXX, LINE_XXOO, LINE_XOOX);
 
+    if ( part == NULL ) {
+        wrefresh(w_details);
+        return;
+    }
```

Comment:
Oh, yes that would have made for a much cleaner change. Never even entered my mind :) Will keep this in mind for future refactoring, thanks!